pennylane_lightning/lightning_gpu/lightning_gpu.py

```diff
@@ -209,14 +244,74 @@ def __init__(
             super().__init__(wires, shots=shots, c_dtype=c_dtype)
 
             self._dp = DevPool()
-            self._sync = sync
-            self._batch_obs = batch_obs
 
-            self._num_local_wires = self.num_wires
-            self._gpu_state = _gpu_dtype(c_dtype)(self._num_local_wires)
+            if not mpi:
+                self._mpi = False
+                self._num_local_wires = self.num_wires
+                self._gpu_state = _gpu_dtype(c_dtype)(self._num_local_wires)
+            else:
+                self._mpi = True
+                self._mpi_init_helper(self.num_wires)
+
+                if mpi_buf_size < 0:
```

Comment:
If 0, does this succeed?

Reply:
yes, it's default as 0 which will assign 64 MiB GPU memory for MPI operations.